src/Ticket.java

```diff
@@ -1,3 +1,5 @@
+//I am not sure about this, but Rick told me that when using a record all objects are the same, but I want
```

Comment:
Comment seems cut off 😉

Reply:
Yeah I know. Sorry for that. 
Rick explained Records to me, and either I misunderstood or explained it incorrectly. I was typing this and then looking up stuff and didn't come back to it.

Could you explain to me the difference between a record and class?
Does it matter at all when it comes to how the object are referenced in an Array(List)?

Reply:
BTW I have totally taken your return suggestion and immediately implemented it

Reply:
Basically, the difference between a record and a class is that the first one is immutable and does not need a lot of boilerplate code (see e.g. [here](https://www.baeldung.com/java-record-keyword#purpose)). 

When referencing both things in e.g. an ArrayList, there's no difference.